# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
posterior_motor:
  ID: 2
  Return_Delay_Time: 2
  #CW_Angle_Limit: 0 #not working
  #CCW_Angle_Limit: 4094 #not working
  


